### PR TITLE
0.9 port, docs

### PR DESCRIPTION
**Pull request by atotic**
Funny story: checked out google-sheets, forgot to switch the branch. I was like "lazy Eric, this code is ancient". 2 hours later, had something working, was going to commit, and realized what an idiot I was.

It made it a lot easier to understand your port, and you did a better job. Did file a bug against ajax, they should not fire on auto if there is no url.

Everything looks good except for the markers. I think this bug is in google-map, the demo is not showing giants logo either. Will look at it next week.

PS: nice gutsy demo, sheets + maps.

### CODE REVIEW

File: google-sheets.html [hunk 1]
```diff
@@ -226,7 +220,7 @@
         value: function() { return []; },
         readOnly: true,
         notify: true,
-        reflectToAttribute: true
+        // reflectToAttribute: true
```
Comment (ebidel) [inline google-sheets.html]: yea, don't think we need this. Not sure why I added it.

**Comment by ebidel** [comment]
Sorry for the troubles :( That's happened to me plenty of times.

Thanks about the demo. I thought it was a good way of show some of this stuff working together. IMO, most our demos don't do that enough.

**Comment by ebidel** [comment]
LGTM. Just remove the commented `reflectToAttribute`

**Comment by atotic** [comment]
Not quite ready yet, found a few more things, working through it.

**Comment by atotic** [comment]
It is working. Got bit by fittomarkers should be fit-to-markers
In the spirit of sane property naming, should we rename tabid to tabId?
Also, spreadsheets are currently not shared accross instances. Should I make them shared, or change the docs?
Fixed reflectTo, and the way openInGoogleDocsUrl was computed.

**Comment by ebidel** [comment]
I started this port before I understood the rules on attribute -> property naming. +1 on the `tabId` switch. It's ok that sheets are not shared across instance. Let's just update the docs accordingly.

**Comment by atotic** [comment]
fixed tabid, removed obsolete demo-private (that code gave me a scare, was I on the wrong branch again?!). Some notes:
clientId will not be necessary once my google-signin lands. I'd remove it, clientId should be specified by the app that uses the component.
Inside demo.html, there are two self-binding templates with two google-sheets accessing the same document. I have local mods to simplify this: one template, one sheet. Do you want this, or are two templates there for future expansion?

**Comment by ebidel** [comment]
We need to keep the demo-private.html. That code just hasn't been ported yet. Don't worry about it. I'll take care of it. 

> clientId will not be necessary 

Did you check with @Zoramite / @addyosmani. Requiring a client id on each element was a design choice we made early on. I believe there was an ask to support multiple clientIds in an app.

The two demos were mention to show how to use the element inside of an auto binding template and outside of it. That was intentional.

**Comment by ebidel** [comment]
Take that back about the two templates in the demo. Feel free to consolidate. The demo has changed a bit since I first wrote it and there's no longer need to have two.

**Comment by atotic** [comment]
templates consolidated in demo/index.html. Restored demo-private.

I've discussed single clientId with @Zoramite, he was ok with the change. Multiple clientIds never worked, and I do not think they can. auth2.init can only be called once, and takes one clientId. If multiple apps were running on the same page, they could share data they have no access to, and that would be bad.

**Comment by ebidel** [comment]
Good point on auth2. For all it adds over v1, it has its own set of limitations.

**Comment by ebidel** [comment]
LGTM. Thanks for cranking this out!

**Pull request merged**